src/validation.h

```diff
@@ -226,6 +230,18 @@ struct MempoolAcceptResult {
 MempoolAcceptResult AcceptToMemoryPool(CChainState& active_chainstate, CTxMemPool& pool, const CTransactionRef& tx,
                                        bool bypass_limits, bool test_accept=false) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+/**
+* Atomically test acceptance of multiple transactions.
+* @param[in]    txns                Group of transactions which may be independent or contain
```

Comment:
I think we should think twice w.r.t independence as the evaluation of _topologically_-independent packages might still have interdependent evaluations.

Let's say your package 1 is replacing in-mempool tx A. Such high-feerate tx A is a replacement blocker for the low-feerate package 2. Evaluating package 1 then package 2 make both of them accepted, the reserve isn't true.

Also it make it harder to reason on package upper bound size, as you might have N independent packages all under descendant limit but collectively far above.

Reply:
Current policy is they can be independent or dependent (although I don't think the user would get much use out of independent ones). Given that RBF is now disabled, hopefully this concern is resolved?